packages/core/package.nls.json

```diff
@@ -241,7 +241,7 @@
     "AWS.walkthrough.gettingStarted.connect": "Connect to AWS",
     "AWS.walkthrough.gettingStarted.changeRegions": "Change AWS Regions",
     "AWS.walkthrough.gettingStarted.setupToolchain": "Configure your toolchain",
-    "AWS.command.codewhisperer.title": "CodeWhisperer Invoke Service",
+    "AWS.command.codewhisperer.title": "Manual trigger inline suggestion",
```

Comment:
It may be well beyond the scope of this PR, though I am curious if should replace all instances of `AWS.command.codewhisperer` to `Amazon.command.amazonQ`??

Reply:
>Only commands are renamed.

only customer facing commands are renamed. 

>all instances of AWS.command.codewhisperer

These are the keys for strings that are not public facing. They are not blocker for new release. 

Eventually they should be renamed

Reply:
Thank you @leigaol for that. It is super helpful :)